jsk_data/node_scripts/data_collection_server.py

```diff
@@ -86,7 +87,10 @@ def __del__(self):
             sub.unregister()
 
     def sub_cb(self, msg, topic_name):
-        self.msg[topic_name] = msg
+        self.msg[topic_name] = {
+                'stamp': msg.header.stamp if msg._has_header else rospy.Time.now(),
```

Comment:
I don't think linter is perfect, and I think it needs human help.
Do you think it is ok if you add 100 spaces over there?
There is no linter error, but it should be fixed, isn't it?